modules/nf-core/kraken2/add/tests/main.nf.test

```diff
@@ -0,0 +1,56 @@
+nextflow_process {
+
+    name "Test Process KRAKEN2_ADD"
+    script "../main.nf"
+    process "KRAKEN2_ADD"
+    tag "kraken2"
+    tag "kraken2/add"
+    tag "untar"
+    tag "modules"
+    tag "modules_nfcore"
+    
+    setup {
+        run("UNTAR") {
+            script "modules/nf-core/untar/main.nf"
+            process {
+                """
+                input[0] = Channel.of([
+                    [],
+                    file(
+                        params.test_data['sarscov2']['metagenome']['prot_accession2taxid_gz'],
+                        checkIfExists: true
+                    )
+                ])
+                """
+            }
+        }
+    }
+
+    test("sarscov2 protein_db") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true)
+                    ]
+                input[1] = file(params.test_data['sarscov2']['metagenome']['prot_names_dmp'], checkIfExists: true)
+                input[2] = file(params.test_data['sarscov2']['metagenome']['prot_nodes_dmp'], checkIfExists: true)
```

Comment:
Does this really work with protein accessions!?

Reply:
> Does this really work with protein accessions!?

It should work, but it should be specified until the build step using the --protein option:

```
--protein                  Build a protein database for translated search
```